src/components/BlogCard/BlogCard.react.js

```diff
@@ -0,0 +1,109 @@
+//@flow
+
+import * as React from "react";
+import Card from "../Card/Card.react";
+import Icon from "../Icon/Icon.react";
+
+type Props = {|
+  +children?: React.Node,
+  +title?: string,
+  +titleUrl?: string,
+  +avatarUrl?: string,
+  +description?: string,
+  +date?: string,
+  +imgUrl?: string,
+  +iconName?: string,
+  +iconHref?: string,
+  +authorName?: string,
+  +avatarImgSrc?: string,
+  +imgSrc?: string,
+  +imgAlt?: string,
+  +aside?: boolean,
+  +postHref?: string,
+  +profileHref?: string,
+|};
+
+function BlogCard({
+  children,
+  title,
+  description,
+  avatarUrl,
+  imgUrl,
+  imgAlt,
+  aside,
+  authorName,
+  date,
+  imgSrc = "",
+  avatarImgSrc = "",
+  iconName,
+  iconHref,
+  postHref,
+  profileHref,
+}: Props): React.Node {
+  return !aside ? (
+    <Card>
+      <a href={postHref}>
+        <img className="card-img-top" src={imgSrc} alt={imgAlt} />
+      </a>
+      <Card.Body className="d-flex flex-column">
+        <h4>
+          {""}
```

Comment:
As far as I'm aware, an empty string will have no effect (feel free to correct me if I'm wrong).  If it has no effect we should probably just remove it.